README.md

```diff
@@ -1,142 +1,220 @@
+[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://github.com/wso2/product-ei/blob/master/LICENSE)
+[![Twitter](https://img.shields.io/twitter/follow/wso2.svg?style=social&label=Follow)](https://twitter.com/intent/follow?screen_name=wso2)
+
+# WSO2 Enterprise Integrator
+
+WSO2 Enterprise Integrator is an open source, fast, cloud native and scalable integration solution which can be 
+positioned as the core of WSO2 Integration Agile Platform. It enables enterprise integration developers to build 
+sophisticated integration solutions, achieving digital agility. As a mature integration product since 2005,
+*(branded as WSO2 ESB at the time)*, it continues to be the most sophisticated and extensible enterprise integration 
+solutions out there in open source.
+
+Actively maintained, with commercial support from WSO2 Inc, WSO2 Enterprise Integrator is widely used in production at 
+companies around the globe starting from startups to fortune 500 companies in the fields of government, healthcare, 
+banking, education, communication, etc.
+
+[Installation](https://docs.wso2.com/display/EI640/Installation+Guide) | 
+[Documentation](https://docs.wso2.com/display/EI640/) | 
+[Mailing Lists](https://wso2.com/mail/) | 
+[Blog](https://wso2.com/blogs/thesource/) | 
+[Support](https://wso2.com/subscription) | 
+[Nightly Builds](https://wso2.org/jenkins/view/products/job/products/job/product-ei/)
+
+
+## Outline 
+
+- [**Why WSO2 Enterprise Integrator?**](#Why-WSO2-Enterprise-Integrator)
+- [**Features**](#Features)
+- [**Distributions**](#Distributions)
+- [**Installation and Run**](#Installation-and-Run)
+- [**Artifact Development**](#Artifact-Development)
+- [**Building from Source**](#Building-from-Source)
+- [**Enterprise Support**](#Enterprise-Support)
+- [**Licence**](#License)
+
+
+## Why WSO2 Enterprise Integrator
+
+WSO2 Enterprise Integrator is an open source, hybrid integration platform. It is comprised of profiles those address different
+parts of a complete integration story. 
+
+**Enterprise Service Bus profile**: If you are trying to interconnect your enterprise applications on-premise, legacy or cloud, WSO2 EI can act as a Service Bus. It can help transforming messages 
+  to different formats, standards, use different protocols to communicate and mediating messages across the applications. 
+  With 100+ ready made cloud connectors, intuitive tooling,  analytic capabilities WSO2 EI provides a greater agility to 
+  meet growing and changing enterprise demands. It can also accelerate bringing your enterprise data to the screen by 
+  exposing them as services, APIs.
+ 
+**Message Broker profile**: WSO2 EI is comprised of asynchronous messaging capability and can act as an AMQP broker. 
+
+**Business-process profile**: If your integration story contains human interactions (i.e approval process) WSO2 EI can 
+  corporate with BPS/BPMN and human task capabilities.
```

Comment:
```suggestion
  cooperate with BPS/BPMN and human task capabilities.
```

What do we mean when we say, "cooperate with BPS"? Someone reading this as a new user will be slightly confused.

Reply:
Agreed. Will change as "WSO2 EI can provide BPS/BPMN and human task capabilities to develop work-flows."